src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -61,11 +64,35 @@ const stayAtHomeOrders = {
   WI: 20200325,
 }
 
+const statePopulations = StatesWithPopulation.features.reduce((acc, cur) => {
+  acc[cur.properties.STUSPS] = cur.properties.population
+  return acc
+}, {})
+
+const territoryPopulations = {
+  GU: 164229,
+  VI: 107268,
+  MP: 55144,
+  AS: 55641,
+}
+
+function sortGroupedData(groupedData) {
+  return groupedData.sort((a, b) => {
+    const lastA = a.values[0]
+    const lastB = b.values[0]
+
+    const lastATotal = calculateTotal(lastA)
```

Comment:
the field `totalTestResults` contains `positive` + `negative` already so using this function is no longer necessary

Reply:
I am not seeing a `totalTestResults` field in this data. Inspecting the raw data:

<img width="593" alt="Screen Shot 2020-04-10 at 6 34 55 PM" src="https://user-images.githubusercontent.com/63169602/79027404-13578f80-7b5a-11ea-9033-d4a0f7312171.png">

Reply:
you'll have to modify the graphql query:
```
  const query = useStaticQuery(graphql`
    {
      allCovidStateDaily {
        edges {
          node {
            date
            state
            positive
            negative
            totalTestResults
          }
        }
      }
    }
  `)
```

Reply:
also checkout http://localhost:8000/___graphql to make it easier to build your queries in the future.

Reply:
Ah, gotcha. Will update and remove the unnecessary calculations.

Reply:
Removed in e13d9ce.